# **1 Sign In**
```Ssh -X rhoadesn@ucrbiocluster.ucr.edu Pw: rhoadesn1```
# **2 Obtaining resources to conduct high computational numbers**
```srun --x11 --mem=24gb --cpus-per-task 20 --ntasks 1 --time 10:00:00 --pty bash -I```
# **3 Load Qiime 2**
```module load qiime2```
# **4 Steps to be clarified for fastq**
# **5 Importing**
```qiime tools import--type'SampleData[PairedEndSequencesWithQuality]' --input-path sequences --source-format CasavaOneEightSingleLanePerSampleDirFmt --output-path demux-paired-end.qza```
demux-paired-end.qza will be produced
# **6 Determines number of sequences were obtained per sample and sequence quality**
Locate qzv file made and drag it to View in Qiime2 website
Two graphs will appear:
Forward reads - 260
Reverse reads – 160
Numbers will vary depending quality of forward and reverse reads
```qiime demux summarize --i-data demux-paired-end.qza --o-visualization demux.qzv```
# **7 DADA2**
Quality control, filter phiX reads, and filter chimeric sequences
Code will depend on the sequence base number in the forward and reverse reads from previous
visual graph. i.e. …–p-trunc-len-f (#here) --ptrunc-len-r (#here)…
```qiime dada2 denoise-paired --i-demultiplexed-seqs demux-paired-end.qza --o-representativesequences rep-seqs-dada2.qza --o-table table-dada2.qza --o-denoising-stats stats-dada2.qza --ptrim-left-f 1 --p-trim-left-r 1 --p-trunc-len-f 278 --p-trunc-len-r 200 --p-n-threads 30```
# **8  FeatureTable and FeatureData summaries** 
information on how many sequences are associated
with each sample and with each feature, histograms of those distributions, and some related
summary statistics
```qiime feature-table summarize --i-table table-dada2.qza --o-visualization table.qzv```
table-dada2.qzv will be produced
# **9 Generate a tree for diversity analysis**
**Part 1**
```qiime alignment mafft --i-sequences rep-seqs-dada2.qza --o-alignment aligned-rep-seqs.qza --p-nthreads 30```
aligned-rep-seqs.qza will be produced  
**Part 2a**
```qiime alignment mask --i-alignment aligned-rep-seqs.qza --o-masked-alignment masked-aligned-repseqs-pe.qza```  
taxa-bar-plots.qzv will be produced  
**Part 2b**
```qiime metadata tabulate --m-input-file taxonomy.qza --o-visualization taxonomy.qzv```  
taxonomy.qzv will be produced
