# *Programs you need:**

**SRA tool kit allows reading and writing of SRA files:**  
```module load sratoolkit```

**Bowtie2 is a short read aligner:**
```module load bowtie2```

**Samtools is a suite of programs for interacting with high-throughput sequencing data:**
```module load samtools```

**BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF.**
```module load bcftools```
  
# **Gain access**

```wget https://sra-download.ncbi.nlm.nih.gov/traces/sra78/SRR/008462/SRR8666009```

```fast-dump SRR8666009```

# **Align SRA reads to TS using the local flag. I think we may need to add an additional step here**

```bowtie2 -q -p 2 -x ./Amuc_TS -U SRR8666009.fastq --quiet --al-gz ./009.Amuc_1100_TS.fastq.gz --local  1> 009_Amuc_1100_TS.sam 2> 009_Amuc_1100_alignment_stats.txt```

# **Sort and reduce the size of the alignment file**

```samtools view -bu 009_Amuc_1100.sam | samtools sort -T 009_Amuc_1100 -o 009_Amuc_1100.sorted.bam -@ 2```

# **Optional step to keep storage requirements down:(only do if you are sure the alignment worked)**

```rm 009_Amuc_1100_TS.sam SRR8666009.fastq```

# **Create an index of your input sequence**

```samtools faidx Amuc_TS.fasta```

# ***!!!RUN ALL SAMPLES BEFORE THE FOLLOWING!!!***

```bcftools mpileup -Ou -f Amuc_TS.fasta *sorted.bam | bcftools call -mv -Ob -o calls.bcf```

```bcftools view calls.bcf | vcfutils.pl varFilter - > my.var-final.vcf```
